news/_posts/2022-01-13-new-theme-announce.md

```diff
@@ -0,0 +1,71 @@
+---
+layout: post
+title: We are working on a new front office theme
+subtitle: Here are some more details about it
```

Comment:
Suggestions
- Let's discuss what we are working on
- Diving into the details

Reply:
@kpodemski already made me change this :/

Reply:
Then I shall have a fair fight with him at the top of a tower.

Reply:
Yes, please film it, it's for a friend

Reply:
@matks 🥊 :D

Reply:
<img src="https://qph.fs.quoracdn.net/main-qimg-97626540ca116eeac7471f8bde83715f.webp">

Reply:
So, what to do?

Reply:
@kpodemski is a tech evangelist, I'm a tech manager. I think he knows better what to write so let's apply his feedback.